sw/device/mask_rom/boot.md

```diff
@@ -269,3 +266,67 @@ peripheral_lockdown() {
   // inputs getting locked is not something the ROM_EXT can choose.
 }
 ```
+
+# Info Structures
+
+There are two main info structures used by the Mask ROM:
+
+*   The Boot Policy structure, used to choose a ROM_EXT to boot.
+*   The ROM_EXT manifest, used to contain information about a specific ROM_EXT.
+
+## Boot Policy Structure
+
+Accessed by:
+
+*   Mask ROM (to choose ROM_EXT, and during bootstrapping to bless new ROM_EXT).
+*   ROM_EXT (during firmware update).
+
+Needs to contain:
+
+*   Identifier (so we know we're reading the right thing)
+
+    This also acts like a version, because the Mask ROM code that parses the
+    boot policy can never be updated. New Mask ROM version means new identifier.
+
+*   Which ROM_EXT slot should be chosen first (2.b, 2.c.i).
+*   What to do if ROM_EXT does not validate (2.c.ii, 2.c.iii):
+    *   Try Alternate ROM_EXT; or
+    *   Fail to Boot
+*   What to do if ROM_EXT validates successfully and boots:
+    *   Do Nothing; or
+    *   Set Current ROM_EXT as Primary
```

Comment:
This is an interesting point - yes the flash will degrade over time (so we should minimize writes), but we also want to avoid conditionals in our Mask ROM code.